### PR TITLE
Create uni-freiburg-geschichte.csl

### PR DESCRIPTION
**Pull request by soenkewindhausen**
Created a citation style after the Proseminarheft des Historischen Seminars.

### CODE REVIEW

**Comment by rmzelle** [comment]
Thanks for your submission. Is there an online style guide available for this style? Per https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link, we typically require each CSL style to include a link to the format it describes.

File: uni-freiburg-geschichte.csl [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+        </if>
+        <else-if type="entry-encyclopedia" match="any">
+          <label suffix=" " variable="locator" form="short"/>
+          <text variable="page"/>
+        </else-if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="access">
+    <text macro="e-resource"/>
```
Comment (POBrien333) [inline uni-freiburg-geschichte.csl]: Delete this as it doesn't exist. (or add it back if you deleted by chance).

File: uni-freiburg-geschichte.csl [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+  <info>
+    <title>Uni Freiburg Geschichte</title>
+    <title-short>Uni Freiburg</title-short>
+    <id>http://www.zotero.org/styles/uni-freiburg-geschichte</id>
+    <link href="http://www.zotero.org/styles/uni-freiburg-geschichte" rel="self"/>
+     <link href="https://mittelalter2.geschichte.uni-freiburg.de/lehre/proseminarheft-mittelalter" rel="documentation"/>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <updated>2020-02-12T14:08:32+00:00</updated>
+  </info>
```
Comment (POBrien333) [inline uni-freiburg-geschichte.csl]: Add this before </info>:
    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>

Comment (soenkewindhausen) [reply uni-freiburg-geschichte.csl]: thanks for the help

**Comment by rmzelle** [comment]
@soenkewindhausen, do you know how this citation format differs from https://www.zotero.org/styles?q=id%3Auniversitat-freiburg-geschichte ?

**Comment by soenkewindhausen** [comment]
> @soenkewindhausen, do you know how this citation format differs from https://www.zotero.org/styles?q=id%3Auniversitat-freiburg-geschichte ?

yes it is quite different. I am not sure who made this, but this is not the style we use in History at this university. (e.g. Date, e-resources, journal names and more)

I don't understand the failed check code ... sorry. But i would love to upload it to help some friends of mine. 
thanks for the help

**Comment by POBrien333** [comment]
It is not validating as you are calling a macro called "access" that doesn't exist. Lines 468 and 493. 

You can always use the Validator to decipher the test reports: validator.citationstyles.org/

**Comment by rmzelle** [comment]
> > @soenkewindhausen, do you know how this citation format differs from https://www.zotero.org/styles?q=id%3Auniversitat-freiburg-geschichte ?
> 
> yes it is quite different. I am not sure who made this, but this is not the style we use in History at this university. (e.g. Date, e-resources, journal names and more)

@glennfed, are you still around? You made some corrections to the "universitat-freiburg-geschichte.csl" CSL style in the past, and I was wondering if you remembered which citation guidelines you were looking at at the time. I'd like to find out whether there are two different citation formats in use within the history department at this university, or whether we should just replace the old CSL style with this new one.

**Comment by rmzelle** [comment]
@soenkewindhausen, if we don't hear back from @glennfed, I'd prefer to just replace the old style with this one (especially since the old style doesn't have a link to any citation style guide). We can easily make that change on our end and update your pull request to modify the existing CSL style instead of creating a new one.

But I'll give it a few more days.

**Comment by soenkewindhausen** [comment]
> @soenkewindhausen, if we don't hear back from @glennfed, I'd prefer to just replace the old style with this one (especially since the old style doesn't have a link to any citation style guide). We can easily make that change on our end and update your pull request to modify the existing CSL style instead of creating a new one.
> 
> But I'll give it a few more days.

Yes sounds good. I went through all the Style guides in the History department and dind't find @glennfed 's version of the Style. 
They are all different. 

Thanks!

**Comment by rmzelle** [comment]
@soenkewindhausen, one more question:

Did you base this on an existing CSL style? If so, we'd like to know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

**Comment by soenkewindhausen** [comment]
> @soenkewindhausen, one more question:
> 
> Did you base this on an existing CSL style? If so, we'd like to know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

Yes I did. I am not 100% sure because I did this a few months ago. But I am pretty sure it was based on: 

id:technische-universitat-dresden-kunstgeschichte-note
https://www.zotero.org/styles?q=id%3Atechnische-universitat-dresden-kunstgeschichte-note

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**